docs/source/management/index.md

```diff
@@ -108,16 +112,104 @@ Available SRE configurations for context 'green':
 └──────────────┴──────────┘
 ```
 
-## Removing a deployed Data Safe Haven
+### Removing a deployed Data Safe Haven
 
 - Run the following if you want to teardown a deployed SRE:
 
 ```{code} shell
 $ dsh sre teardown YOUR_SRE_NAME
 ```
 
+::::{admonition} Tearing down an SRE deletes ALL data and resources
+Running `dsh sre teardown` will remove all resources deployed within the SRE.
+Ensure that any desired outputs have been egressed before deleting the SRE.
+Any data remaining on the SRE will be deleted.
+The user groups for the SRE on Microsoft Entra ID will also be deleted.
+::::
+
 - Run the following if you want to teardown the deployed SHM:
 
 ```{code} shell
 $ dsh shm teardown
 ```
+
+::::{admonition} Tearing down an SHM
+All SREs associated with the SHM must be torn down before the SHM can be torn down.
+Tearing down the SHM deletes any remotely stored SRE configurations.
+::::
+
+## Managing data ingress and egress
+
+### Data Ingress
+
+It is the data provider's responsibility to upload the data required by the safe haven.
+
+```{important}
+Any data ingress must be signed off by the {ref}`role_data_provider_representative`, {ref}`role_investigator` and referee (if applicable).
+```
+
+The following steps show how to generate a temporary write-only upload token that can be securely sent to the data provider, enabling them to upload the data:
+
+- In the Azure portal select `Subscriptions` then navigate to the subscription containing the relevant SHM
+- Search for the resource group: `shm-<SHM ID>-sre-<SRE ID>-rg`, then click through to the storage account called: `sh<first three letter of SHM ID>sre<first three letters of SRE ID>sensitivedata`
+- Click `Networking` under `Settings` and ensure that the data provider's IP address is one of those allowed under the `Firewall` header, then hit the save icon in the top left
```

Comment:
We have been using the [`guilabel` role](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-guilabel) for buttons in the documentation.

For example,

https://github.com/alan-turing-institute/data-safe-haven/blob/d2141434b496c5863ff8065537ebf56cbd78ef62/docs/source/deployment/configure_entra_id.md?plain=1#L45-L50